get_forecast.py

```diff
@@ -131,19 +143,36 @@ def send_sns(boto3_session, sns_arn, message):
 
 
 def display_output(boto3_session, message):
-    secrets_manager_client = boto3_session.client('secretsmanager')
+    secrets_manager_client = None
+    secret = ""
+    if 'SECRETS_MANAGER_ENDPOINT' in os.environ:
+        logger.info("Using secrets_manager_endpoint.")
+        secrets_manager_client = boto3_session.client('secretsmanager', os.environ['SECRETS_MANAGER_ENDPOINT'])
+    else:
+        secrets_manager_client = boto3_session.client('secretsmanager')
+
+    try:
+        secret = get_secret(secrets_manager_client)
+    except Exception as e:
+        logger.error("get_secret failed")
```

Comment:
isnt this a fatal error
1. logger.error("get_secret failed") should be logger.debug
2. should this be fatal and raise e?